### PR TITLE
Media

### PR DESCRIPTION
**Pull request by m6kang**
Media Section

### CODE REVIEW

File: source/js/media.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        element.style.display = 'none';
+    } else {
+        element.style.display = displayType;
+    }
+}
+
+function youtubeUpload(link) {
+    let youtubeDiv = document.createElement('div');
+    youtubeDiv.setAttribute('class', 'media-item');
+    let youtubeIfr = document.createElement('iframe');
```
Comment (hexalellogram) [inline source/js/media.js:26]: Codacy has a fix for the issue: ['youtubeIfr' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const youtubeIfr = document.createElement('iframe');
```

File: source/js/media.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    youtubeIfr.setAttribute('src', `https://www.youtube.com/embed/${link.substring(32)}`);
+    youtubeIfr.style.width = '560';
+    youtubeIfr.style.height = '315';
+    youtubeDiv.appendChild(youtubeIfr);
+    mediaDiv.appendChild(youtubeDiv);
+}
+
+function pinterestUpload(link) {
+    let pinterestDiv = document.createElement('div');
```
Comment (hexalellogram) [inline source/js/media.js:35]: Codacy has a fix for the issue: ['pinterestDiv' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const pinterestDiv = document.createElement('div');
```

File: source/js/media.js [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    pinterestIfr.style.width = '236';
+    pinterestIfr.style.height = '520';
+    pinterestIfr.setAttribute('scrolling', 'no');
+    pinterestIfr.setAttribute('frameborder', '0');
+    pinterestDiv.appendChild(pinterestIfr);
+    mediaDiv.appendChild(pinterestDiv);
+}
+
+function soundcloudUpload(link) {
+    let soundcloudDiv = document.createElement('div');
```
Comment (hexalellogram) [inline source/js/media.js:49]: Codacy has a fix for the issue: ['soundcloudDiv' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const soundcloudDiv = document.createElement('div');
```

File: source/js/media.js [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    youtubeIfr.style.height = '315';
+    youtubeDiv.appendChild(youtubeIfr);
+    mediaDiv.appendChild(youtubeDiv);
+}
+
+function pinterestUpload(link) {
+    let pinterestDiv = document.createElement('div');
+    pinterestDiv.setAttribute('class', 'media-item');
+    let pinterestIfr = document.createElement('iframe');
+    let pinterestUrl = `https://assets.pinterest.com/ext/embed.html?id=${link.substring(30)}`;
```
Comment (hexalellogram) [inline source/js/media.js:38]: Codacy has a fix for the issue: ['pinterestUrl' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const pinterestUrl = `https://assets.pinterest.com/ext/embed.html?id=${link.substring(30)}`;
```

File: source/js/media.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+    youtubeIfr.style.width = '560';
+    youtubeIfr.style.height = '315';
+    youtubeDiv.appendChild(youtubeIfr);
+    mediaDiv.appendChild(youtubeDiv);
+}
+
+function pinterestUpload(link) {
+    let pinterestDiv = document.createElement('div');
+    pinterestDiv.setAttribute('class', 'media-item');
+    let pinterestIfr = document.createElement('iframe');
```
Comment (hexalellogram) [inline source/js/media.js:37]: Codacy has a fix for the issue: ['pinterestIfr' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const pinterestIfr = document.createElement('iframe');
```

File: source/js/media.js [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+    pinterestIfr.setAttribute('scrolling', 'no');
+    pinterestIfr.setAttribute('frameborder', '0');
+    pinterestDiv.appendChild(pinterestIfr);
+    mediaDiv.appendChild(pinterestDiv);
+}
+
+function soundcloudUpload(link) {
+    let soundcloudDiv = document.createElement('div');
+    soundcloudDiv.setAttribute('class', 'media-item');
+    let soundcloudIfr = document.createElement('iframe');
```
Comment (hexalellogram) [inline source/js/media.js:51]: Codacy has a fix for the issue: ['soundcloudIfr' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const soundcloudIfr = document.createElement('iframe');
```

File: source/js/media.js [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+    const element = document.getElementById(elementID);
+    if (element.style.display === displayType) {
+        element.style.display = 'none';
+    } else {
+        element.style.display = displayType;
+    }
+}
+
+function youtubeUpload(link) {
+    let youtubeDiv = document.createElement('div');
```
Comment (hexalellogram) [inline source/js/media.js:24]: Codacy has a fix for the issue: ['youtubeDiv' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const youtubeDiv = document.createElement('div');
```

File: source/js/media.js (hunk 4 above)
Comment (hexalellogram) [inline source/js/media.js:38]: Codacy has a fix for the issue: ['pinterestUrl' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const pinterestUrl = `https://assets.pinterest.com/ext/embed.html?id=${link.substring(30)}`;
```

File: source/js/media.js (hunk 2 above)
Comment (hexalellogram) [inline source/js/media.js:35]: Codacy has a fix for the issue: ['pinterestDiv' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const pinterestDiv = document.createElement('div');
```

File: source/js/media.js (hunk 7 above)
Comment (hexalellogram) [inline source/js/media.js:24]: Codacy has a fix for the issue: ['youtubeDiv' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const youtubeDiv = document.createElement('div');
```

File: source/js/media.js (hunk 1 above)
Comment (hexalellogram) [inline source/js/media.js:26]: Codacy has a fix for the issue: ['youtubeIfr' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const youtubeIfr = document.createElement('iframe');
```

File: source/js/media.js (hunk 6 above)
Comment (hexalellogram) [inline source/js/media.js:51]: Codacy has a fix for the issue: ['soundcloudIfr' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const soundcloudIfr = document.createElement('iframe');
```

File: source/js/media.js (hunk 3 above)
Comment (hexalellogram) [inline source/js/media.js:49]: Codacy has a fix for the issue: ['soundcloudDiv' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const soundcloudDiv = document.createElement('div');
```

File: source/js/media.js (hunk 5 above)
Comment (hexalellogram) [inline source/js/media.js:37]: Codacy has a fix for the issue: ['pinterestIfr' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/cse110-sp21-group31/cse110-sp21-group31/pullRequest?prid=7577486)
```suggestion
    const pinterestIfr = document.createElement('iframe');
```

**Comment by hexalellogram** [comment]
Can we commit these Codacy fixes that are being suggested? Or not yet?

**Review by artyom2001 — APPROVED** [review]
Media section is now fully functional! Also fixes styling for the clock button. Good to merge!

**Pull request merged**